eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/entity/SequenceEntity.java

```diff
@@ -81,7 +90,16 @@ public ExtendedPhenotype getPrimaryExtendedPhenotype() {
 	public void setPrimaryExtendedPhenotype(ExtendedPhenotype inExtendedPhenotype) {
 		primaryExtendedPhenotype = inExtendedPhenotype;
 	}
+        
+        
+	public List<ExtendedPhenotype> getExtendedPhenotypes() {
+		return extendedPhenotypes;
+	}
 
+	public void setExtendedPhenotypes(List<ExtendedPhenotype> inExtendedPhenotypes) {
```

Comment:
Do we need the extendedPhenotype getters and setters? If you call setRelations(), getExtendedPhenotypes() won't return an updated list until you refresh the entity. Also, setExtendedPhenotypes() doesn't seem very useful.